### DISCUSSION TITLE
Gruppe kann nicht erstellt werden

### DISCUSSION THREAD

**Discussion by hansipete** [OpenCCU/OpenCCU#2915] [🇩🇪 Fragen&Antworten] [answered]
### Describe the issue you are experiencing

Neue Gerätegruppe (BidCos) kann nicht erstellt werden. "Gruppe wird gepeichert" Overlay kommt und bleibt für immer (15min getestet)

Habe mich neu eingeloggt und geschaut ob die Gruppe ggf. doch erstellt wurde. Leider nicht. 

Es macht keinen Unterschied ob die Gruppe leer oder befüllt gespeichert wird.

Neue HomematicIP Gruppe funktioniert einwandfrei (nur leer getestet, weil ich keine IP verwende)

### Describe the behavior you expected

Gruppe sollte innerhalb weniger Sekunden gespeichert werden, oder eine Fehlermeldung kommen.

### Steps to reproduce the issue

1. Einloggen als "Admin"
2. Einstellungen > Gruppen
3. Neu → Gruppentyp: "Heizungssteuerung"
...

### What is the version this bug report is based on?

3.79.6.20241031

### Which base platform are you running?

ova (Open Virtual Infrastructure, AMD64/x86_64)

### Which HomeMatic/homematicIP radio module are you using?

RPI-RF-MOD

### Anything in the logs that might be useful for us?

```txt
(nicht auffälliges)
```

### Additional information

Fehler in der Browser Konsole (Chrome u. Firefox getestet) direkt nach Click auf "OK" (Speichern)

``` 
VM1624:380 Uncaught TypeError: Cannot read properties of undefined (reading 'id')
    at <anonymous>:380:72
    at Object.p (knockout.js:8:195)
    at _SaveGroup (<anonymous>:379:18)
    at SaveGroup (<anonymous>:354:16)
    at HTMLDivElement.onclick (index.htm?sid=@WiwgFQoYe4@:1:1)
```

**Comment by Baxxy13** [comment, marked as answer]
Liegt eventuell noch ein Gerät im Posteingang rum?
Dann diese erstmal rausholen und erneut versuchen.

**Comment by hansipete** [comment]
Das klappt! Danke für die schnelle Hilfe.

**Reply by Baxxy13** [reply]
Sehr gut.
Du bist da in einen lange bestehenden Bug gelaufen der aber mit der nächsten RaspberryMatic behoben sein wird. (ist er bereits in den Nightly Snapshots)
